L1Trigger/ME0Trigger/plugins/ME0TriggerProducer.cc

```diff
@@ -0,0 +1,47 @@
+#include "L1Trigger/ME0Trigger/plugins/ME0TriggerProducer.h"
+#include "L1Trigger/ME0Trigger/src/ME0TriggerBuilder.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DataFormats/GEMDigi/interface/ME0TriggerDigiCollection.h"
+#include "Geometry/Records/interface/MuonGeometryRecord.h"
+#include "Geometry/GEMGeometry/interface/ME0Geometry.h"
+
+ME0TriggerProducer::ME0TriggerProducer(const edm::ParameterSet& conf) 
+{
+  me0PadDigiProducer_ = conf.getParameter<edm::InputTag>("ME0PadDigiProducer");
+
+  trigBuilder_.reset( new ME0TriggerBuilder(conf) ); // pass on the conf
```

Comment:
Okay. Somewhat off-topic for this PR, but there are a couple of open issues related to that topic:
#16218
https://github.com/cms-sw/cmssw/issues/16121#issuecomment-291515300